pkg/csi/updatedbconf/register.go

```diff
@@ -28,6 +28,12 @@ func Register(_ manager.Manager, cfg config.Config) error {
 		glog.Info("/etc/updatedb.conf has no changes, skip updating")
 		return nil
 	}
+	glog.Info("backup old /etc/updatedb.conf to /etc/updatedb.conf.backup")
```

Comment:
I think we need check if /etc/updatedb.conf is without any changes then backup it.

Reply:
Line27 has already done this? 
If config file has no change then return.

Reply:
Maybe I mislead you. I think  we should backup the version before doing any modifications by fluid in /etc/updatedb.conf(the original version).

Reply:
Yes, the variable `content` is read from the original version of file `/etc/updatedb.conf` without any change

Reply:
> Yes, the variable `content` is read from the original version of file `/etc/updatedb.conf` without any change

Do you plan to fix it in this PR?